CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-### 0.3.7 (Next)
+### 0.3.8 (Next)
 * Your contribution here.
 
+### 0.3.7 (Next)
```

Comment:
Individual pull requests should not update version numbers as each new version could have multiple changes. Depending on the activity, a release may end up with just one change, but it is not something we decide on as part of a review process.

Reply:
I apologize. I was trying to follow https://github.com/ashkan18/graphlient/pull/59. I will take it out.